README.md

```diff
@@ -1,39 +1,52 @@
 # AWS IoT Device SDK for Java v2
```

Comment:
We need to add the comment about cost disclamers when using samples. Maybe that belongs in the main Samples landing page tho.

Reply:
nvm, that belongs in the samples landing page, not here =D